modules/tinymce/src/core/main/ts/util/Quirks.ts

```diff
@@ -513,9 +512,9 @@ const Quirks = function (editor: Editor): Quirks {
    * WebKit will produce DIV elements here and there by default. But since TinyMCE uses paragraphs by
    * default we want to change that behavior.
    */
-  const setDefaultBlockType = function () {
-    if (settings.forced_root_block) {
-      editor.on('init', function () {
+  const setDefaultBlockType = () => {
+    if (Settings.getForcedRootBlock(editor)) {
```

Comment:
If this had been my change. getForcedRootBlock already existed, and I thought you'd be displeased with that and a hasForcedRootBlock.

Reply:
I don't follow sorry... this is your change? and in `modules/tinymce/src/core/main/ts/keyboard/CefNavigation.ts` you replaced this line with the new `hasForcedRootBlock` function you created in this PR. So how you've changed `editor.settings.forced_root_block` is inconsistent.

Reply:
Because I didn't realize it already had a getForcedRootBlock. It is inconsistent, I will give you that.
Which do we prefer, 'hasForcedRootBlock' and 'getForcedRootBlock' or only the later?

Reply:
I commented above already, but I'd suggest keeping `hasForcedRootBlock` and using that here instead.